config/config.go

```diff
@@ -77,8 +77,9 @@ type Hedera struct {
 }
 
 type Handler struct {
```

Comment:
Wouldn't it be better to name the handler `BurnEvent`? The other handlers are `CryptoTransfer` and `ConsensusMessage` -> they represent the event which we are handling